src/app/tests/suites/certification/Test_TC_DGGEN_2_1.yaml

```diff
@@ -47,7 +47,14 @@ tests:
     - label: "Step 1a: Reset Devices to factory defaults"
       PICS: PICS_USER_PROMPT
       verification: |
-          Reset Devices to factory defaults
+          To perform Factory reset on DUT(all-clusters-app)  follow the below step in RPI platform , Pls use equivalent command on the respective platform
```

Comment:
Please use complete words.  No "Pls".  This is a problem in multiple places.

Reply:
Updated.